docs/Troubleshooting/MySqlSizeIssue.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Maintenance Troubleshooting"
+sidebar_position: 4
+description: >
+  Maintenance Troubleshooting
+---
+
+### How to delete old MySQL binlog files?
+
+1. Connect to your MySQL server using the MySQL client or any other database management tool such as PhpMyAdmin, MySQL Workbench, etc.
+
+2. Run the following command to check the current status of your binary log files:
+
+```sql
+SHOW BINARY LOGS;
+```
+This will display a list of all the binary log files that are currently available on your MySQL server.
+
+3. Determine the last binary log file that you want to keep. This is the file that you want to retain for any future point-in-time recovery or replication purposes.
+
+4. Run the following command to purge all binary logs that are older than the binary log file that you want to retain:
+
+```sql
+PURGE BINARY LOGS TO 'binary_log_file_name';
+```
+
+Replace binary_log_file_name with the name of the last binary log file that you want to keep.
+
+5. After running the command, MySQL will delete all binary log files that are older than the specified file. You can verify that the purge was successful by running the SHOW BINARY LOGS; command again.
+
+Note: Keep in mind that deleting old binary log files can affect point-in-time recovery and replication capabilities, so it's important to only delete files that are no longer needed.
```

Comment:
Shall we also mention that they can "automate" this process away by setting the `expire_logs_days` in the MySQL configuration file?

Reply:
Ok, and we will also add it to docker-compose.yaml